infrastructure/tari_script/src/script.rs

```diff
@@ -537,6 +539,19 @@ impl TariScript {
 
         Ok(sig_set.len() == m)
     }
+
+    fn handle_to_ristretto_point(&self, stack: &mut ExecutionStack) -> Result<(), ScriptError> {
+        let item = stack.pop().ok_or(ScriptError::StackUnderflow)?;
+        let scalar = match &item {
+            StackItem::Hash(hash) => hash.as_slice(),
+            StackItem::Scalar(scalar) => scalar.as_slice(),
+            _ => return Err(ScriptError::IncompatibleTypes),
+        };
+        let scalar = RistrettoSecretKey::from_bytes(scalar).map_err(|_| ScriptError::InvalidData)?;
+        let ristretto_pk = RistrettoPublicKey::from_secret_key(&scalar);
```

Comment:
May improve readability.

```suggestion
        let ristretto_sk = RistrettoSecretKey::from_bytes(scalar).map_err(|_| ScriptError::InvalidData)?;
        let ristretto_pk = RistrettoPublicKey::from_secret_key(&ristretto_sk);
```

Reply:
Will change :ok_hand: - the RistrettoSecret key part is more of an intermediate step so not technically a secret